src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -671,7 +674,8 @@ private static void addSslConfigs(ConfigDef configDef) {
         SSL_GROUP,
         ++order,
         Width.SHORT,
-        SECURITY_PROTOCOL_DISPLAY
+        SECURITY_PROTOCOL_DISPLAY,
+        new EnumRecommender<>(SecurityProtocol.class)
```

Comment:
we can also replace all of the validators with this recommender since it implements `Validator`